iib/web/models.py

```diff
@@ -697,7 +704,12 @@ def index_image_id(cls):
     @declared_attr
     def index_image(cls):
         """Return the relationship to the built index image."""
-        return db.relationship('Image', foreign_keys=[cls.index_image_id], uselist=False)
+        return db.relationship('Image', foreign_keys=[cls.index_resolved_id], uselist=False)
```

Comment:
Does not this change current behavior? Since it was returning `cls.index_image_id` but now it is `cls.index_resolved_id`?

Reply:
```suggestion
        return db.relationship('Image', foreign_keys=[cls.index_image_id], uselist=False)
```